src/TrackGenerator.cpp

```diff
@@ -1269,7 +1330,7 @@ void TrackGenerator::segmentize() {
 
     /* Loop over all Tracks */
     for (int i=0; i < _num_azim; i++) {
-#pragma omp parallel for firstprivate(track)
```

Comment:
Nope, just unnecessary firstprivate. No need to initialize track to an uninitialized pointer.